experiments/compositions/composition/dev/tasks/generate-proto

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd ${REPO_ROOT}/experiments/compositions/composition
+
+BUILDDIR=$(pwd)/.build/
+BINDIR=${BUILDDIR}/bin/
+mkdir -p ${BINDIR}
+
+# Download protoc
+wget -N -O ${BUILDDIR}/protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v27.0/protoc-27.0-linux-x86_64.zip
```

Comment:
Using `@latest` for Go tools can lead to non-reproducible builds, as a new version of the tool could be released at any time with breaking changes. It's better to pin to a specific version to ensure that the build is consistent over time. For example: `go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.34.1`